Add accordions to eligibility page

bkjohnson opened the pull request:
## Description

Related to #16482 

This changes the `verify-eligibility` page so that the `<AdditionalInfo>` components and additional content are removed from the radio buttons and replaced with an accordion above the field.


## Testing done

browser :eyes: 


## Screenshots

![image](https://user-images.githubusercontent.com/2008881/112760899-7af62d00-8fad-11eb-9713-67884dfbb802.png)



### Eligible Veterans

![image](https://user-images.githubusercontent.com/2008881/112760916-89dcdf80-8fad-11eb-86bd-0b60289f2eb7.png)

### Eligible others

![image](https://user-images.githubusercontent.com/2008881/112760925-94977480-8fad-11eb-938e-786e9044df05.png)


### Not eligible

![image](https://user-images.githubusercontent.com/2008881/112760939-9feaa000-8fad-11eb-8b28-41750891c95c.png)


## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

bkjohnson committed:
Add accordions to eligibility page

bkjohnson committed:
Merge branch 'master' into covid-eligibility

stephenBarrs reviewed (approved):
LGTM!

artsymartha68 commented:
1. Accordions should be above “Which of these best describes you?”
2. Accordions should have a H3 header: Learn about who’s eligible for a COVID-19 vaccine at VA

bkjohnson committed:
Update beginning text

- Change accordion item header level
- Add in a h2
- Shift the text around a bit
